### PR TITLE
Make helpers examples runnable

### PR DESCRIPTION
**Pull request by iuliaferoli**
adding examples of how to import the helpers;
adding the client definition since ```bulk``` and ```es``` were used
without context / definition
changed from es to client to stay consistent with the quickstart naming convention

### CODE REVIEW

File: docs/sphinx/helpers.rst [hunk 1]
```diff
@@ -5,6 +5,15 @@ Helpers
 
 Collection of simple helper functions that abstract some specifics of the raw API.
 
+Connecting
+----------
+
+.. code-block:: python
+
+    from elasticsearch import Elasticsearch, helpers
```
Comment (pquentin) [inline docs/sphinx/helpers.rst:13]: Any reason to import helpers here? Do we use it anywhere?

**Review by pquentin — COMMENTED** [review]
Great idea, thanks!

File: docs/sphinx/helpers.rst (hunk 1 above)
Comment (iuliaferoli) [reply docs/sphinx/helpers.rst:13]: We don't; I just wanted to have at least one example of importing the rest of them, other than bulk?
But we can also skip this one

Comment (pquentin) [reply docs/sphinx/helpers.rst:13]: Yes it seems weird if we don't use. Either we call `helpers.bulk` or we drop this IMO.

Comment (iuliaferoli) [reply docs/sphinx/helpers.rst]: fixed

File: docs/sphinx/helpers.rst [hunk 2]
```diff
@@ -5,6 +5,15 @@ Helpers
...
 Collection of simple helper functions that abstract some specifics of the raw API.
 
+Connecting
+----------
+
+.. code-block:: python
+
+    from elasticsearch import Elasticsearch
+    
+    client = Elasticsearch(cloud_id="YOUR_CLOUD_ID", api_key="YOUR_API_KEY")
```
Comment (pquentin) [inline docs/sphinx/helpers.rst:15]:
```suggestion
    client = Elasticsearch("https://.../", api_key="YOUR_API_KEY")
```

**Review by pquentin — APPROVED** [review]

**Pull request merged**